src/scenes/Header/ProfileToolbar/ProfileToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+import { makeStyles, Typography } from '@material-ui/core';
+import { AccountCircle, MoreVert, NotificationsNone } from '@material-ui/icons';
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+import * as React from 'react';
+import { useSession } from '../../../components/Session/Session';
+import { ProfileMenu } from '../ProfileMenu/ProfileMenu';
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  name: {
+    fontWeight: typography.fontWeightMedium,
+  },
+  menuWrap: {
+    position: 'relative',
+  },
+  infoWrap: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    backgroundColor: 'white',
+    boxShadow: '0px 9px 12px rgba(209, 218, 223, 0.43)',
```

Comment:
Let's use `Card`/`Paper`'s `elevation` property instead of this. It isn't going to match directly to the designs, but it will be close. Later we can change this in the theme if we want.

I think we've already customized `Card` so it defaults to the closest one.
https://github.com/SeedCompany/cord-field/blob/439d795856c0528c94fb3eed4c829f1f02856344/src/theme/overrides.ts#L5